application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,35 @@
+Performance indicators showing how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 19 July 2024.
+
+## Availability
+
+### 89 datasets on the platform
+
+**00 national datasets** available from **00 organisations**. 0 added last month (+/-0%).
+
+**00 datasets** provided by **00 local planning authorities**. 0 provided last month (+/-0%).
+
+0 new authorities providing data. +/-0% change from last week.
+
+## Quality
+
+### 00 out of 00 quality score
+
+00 datasets with no issues (0%). 00 fixed last week (+/-0%).
+
+00 datasets conform to the specifications (0%). 00 improved last week (+/-0%).
+
+00 datasets up to date (0%). 00 updated last week (+/-0%).
+
+## Coverage
+
+### 00% nationwide coverage
+
+00% average coverage per dataset. +/-0% change from last week.
```

Comment:
I think these two detailed points are switching between geographic and dataset coverage, so could that be made clearer with something like:

* “00% average nationwide coverage per dataset (XX / 311 LPAs)
* “00% average dataset coverage per LPA (XX / 8 datasets)

Reply:
Yeah, this gets at what we're trying to show a lot better. Thanks!